utils/build-script-impl

```diff
@@ -2444,10 +2444,11 @@ for host in "${ALL_HOSTS[@]}"; do
                     # problematic for overwriting/stripping symbols. Thus, write
                     # permission is added here.
                     for lib_name in tensorflow tensorflow_framework; do
-                        lib="lib${lib_name}.so"
+                        lib="lib${lib_name}.so*"
```

Comment:
@eaplatanios I have incorporated these changes into https://github.com/apple/swift/pull/23895 
Thanks!

Reply:
Thanks @bgogul ! I just pushed one more fix btw.

Reply:
Yes, I noticed adding '-o' caused it to fail. :) I will incorporate your fixes. Thanks again!

Reply:
No problem! :)